app/components/Settings/Preview/CodeEditorField.react.js

```diff
@@ -56,7 +56,8 @@ class CodeEditorField extends Component {
             });
 
             updatePreview({
-                codeSchema: tables
+                codeSchema: tables,
+                code: value
```

Comment:
I'm not sure if I understand this flow. We're already updating the code with `this.props.onChange`. I don't see where `this.props.preview.code` is being used anywhere else

Reply:
Now, we are supplying default queries via `code` prop on Editor Component initial render. `onChange` is not called if we do not edit the query, but directly hit `Run` after loading the ta..  this ensures that the `preview.code` is set to the default query (earlier it was null)..